### PR TITLE
About Conservancy and CWL leadership team

### PR DESCRIPTION
**Pull request by stain**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -257,6 +255,9 @@ Conduct](https://github.com/common-workflow-language/common-workflow-language/bl
 * Ward Vandewege, Curoverse Inc. / Arvados; https://orcid.org/0000-0002-2527-6949
 * Alexander Wait Zaranek, Curoverse Inc. / Arvados; https://orcid.org/0000-0002-0415-9655
 
+
+<a name="Leadership_Team"></a>
```
Comment (mr-c) [inline README.md]: Anchor are auto-created: http://www.commonwl.org/#CWL_Leadership_Team

File: README.md [hunk 2]
```diff
@@ -18,11 +18,9 @@ meet the needs of data-intensive science, such as Bioinformatics, Medical
...
-CWL is developed by an informal, multi-vendor working group consisting of
+CWL is developed by a multi-vendor working group consisting of
 organizations and individuals aiming to enable scientists to share data
-analysis workflows.  [The CWL project is on Github](https://github.com/common-workflow-language/common-workflow-language) and we follow the [Open-Stand.org principles for collaborative open standards development](https://open-stand.org/about-us/principles/)
-
-
+analysis workflows.  [The CWL project is maintained on Github](https://github.com/common-workflow-language/common-workflow-language) and we follow the [Open-Stand.org principles for collaborative open standards development](https://open-stand.org/about-us/principles/). Legally CWL is a member project of [Software Freedom Conservancy](https://sfconservancy.org/news/2018/apr/11/cwl-new-member-project/) and is formally managed by the elected [CWL leadership team](#Leadership_Team), however every-day project decisions are made by the [CWL community](#Support) which is open for participation by anyone.
```
Comment (mr-c) [inline README.md]: Thanks for this! Can we get some line wrapping?

File: README.md (hunk 1 above)
Comment (mr-c) [reply README.md]: Nevermind, this is for constant anchor naming between GitHub and commonwl.org

Comment (stain) [reply README.md]: Yes, I used the HTML version at first, but realized GitHub likes - instead of _. Could do absolute URI though.

**Pull request merged**